Projects/hCoreUI/Sources/hForm/hFieldBackground.swift

```diff
@@ -24,7 +25,11 @@ struct hFieldBackgroundModifier: ViewModifier {
                 hSurfaceColor.Opaque.secondary
             }
         } else {
-            hSurfaceColor.Opaque.primary
+            if useTransparentColor {
+                hFillColor.Opaque.white
```

Comment:
Is this really transparent colour or just white? I guess we want transparent colour?

Reply:
@sladan-hedvig the thing is that it looks like this when I add translucent color, so that's why I went with Opaque

![simulator_screenshot_E34BAADC-454F-4B61-BD22-A1E0EE804E99](https://github.com/user-attachments/assets/a7ea20b8-fcd1-4a57-9f15-684361708a80)